logicalRequirements.md

```diff
@@ -285,6 +285,33 @@ __Additional considerations__
 
 Please note that fulfilling this logical element requires interaction with the door in the adjacent room to be possible (so no active locks on it, and fulfilling its interaction requirements). Fulfilling this logical element also causes the room to be reset, which means all obstacles respawn.
 
+#### adjacentJumpway object
+An `adjacentJumpway` object represents the need for Samus to be able to jump into the room from a door frame or platform in an adjacent room. It has the following properties:
```

Comment:
It is intended that we'll eventually include jumpways for horizontal transitions as well, but that would be a different type of jumpway (e.g. "platformToSide" instead of "doorFrameBelow" or "platformBelow"). Added another sentence to clarify that only vertical transitions are supported for now